providers/implementations/signature/eddsa.c

```diff
@@ -195,6 +230,11 @@ int ed448_digest_verify(void *vpeddsactx, const unsigned char *sig,
     if (!ossl_prov_is_running() || siglen != ED448_SIGSIZE)
         return 0;
 
+#ifdef S390X_EC_ASM
+    if (OPENSSL_s390xcap_P.pcc[1] & S390X_CAPBIT(S390X_SCALAR_MULTIPLY_X448))
```

Comment:
The following addiutional checks are needed (analogous to Ed25519):
```
        && OPENSSL_s390xcap_P.kdsa[0] & S390X_CAPBIT(S390X_EDDSA_SIGN_ED448)
        && OPENSSL_s390xcap_P.kdsa[0]
           & S390X_CAPBIT(S390X_EDDSA_VERIFY_ED448)
```

Reply:
Done..